### ISSUE TITLE
Multiple categories are escaped to one

### ISSUE THREAD

**Issue by nickvergessen** [Heymdall/vcard#5]
The category property has a comma separated value, but it looks like the `,` is escaped and therefor stored as one group `group1\,group2` instead of `group1` and `group2`

https://github.com/Heymdall/vcard/blob/master/src/vcard.js#L162

Fix by replacing the function with:
```js
        var escapeCharacters = function (v, k) {
            if (typeof v === 'undefined') {
                return '';
            }
            v = v
                .replace(/\n/g, '\\n')
                .replace(/;/g, '\\;');

            if (typeof k === 'undefined' || k !== 'categories') {
                return v
                    .replace(/\n/g, '\\n')
                    .replace(/;/g, '\\;')
                    .replace(/,/g, '\\,');
            }
            return v
                .replace(/\n/g, '\\n')
                .replace(/;/g, '\\;');
        };
```

And adding the key to the escape method in:
```js
              if (typeof value.value === 'string') {
                    line += escapeCharacters(value.value);
                } else {
                    // complex values
                    line += value.value.map(function (item) {
                        return escapeCharacters(item,);
                    }).join(';');
                }
```

Not sure if there are more values which are treaten wrong, but I found at least those.

**Comment by Heymdall** [comment]
Can you check [this branch](https://github.com/Heymdall/vcard/tree/fix/split-comma-values)? As I understand it will fix your problems, I've added support for some comma-separated value. So check it out, and let me know if it fixes you problem.

**Comment by nickvergessen** [comment]
Works :+1:

**Comment by Heymdall** [comment]
Closed in version [0.2.6](https://github.com/Heymdall/vcard/releases/tag/v0.2.6)
